test/transforms.jl

```diff
@@ -680,6 +680,49 @@
     @test n1 == n2
   end
 
+  @testset "StdNames" begin
+    # camel, snake, upper tests
+    @test TableTransforms._camel("apple banana") == "AppleBanana"
+    @test TableTransforms._snake("apple banana") == "apple_banana"
+    @test TableTransforms._upper("apple banana") == "APPLEBANANA"
+
+    @test TableTransforms._camel("apple_banana") == "Apple_banana"
+    @test TableTransforms._snake("apple_banana") == "apple_banana"
+    @test TableTransforms._upper("apple_banana") == "APPLE_BANANA"
+
+    @test TableTransforms._camel("apple_Banana") == "Apple_Banana"
+    @test TableTransforms._snake("apple_Banana") == "apple_banana"
+    @test TableTransforms._upper("apple_Banana") == "APPLE_BANANA"
```

Comment:
I think we should organize these tests differently. For example, consider the camel case. Your would like to write a list of strings like `["apple banana", "Apple_banana", ...]` and then apply the camel case function to all elements of the list to make sure all results are equal to `"AppleBanana"`.

Reply:
@juliohm I am trying something like this now, but I came across an edge case. What should `TableTransforms._snake("_A")` return? Do we strip spaces and underscores or not?